common/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+use strum::{Display, EnumString, EnumVariantNames};
+
+#[derive(Clone, EnumString, EnumVariantNames, Display)]
+pub enum Backend {
+    #[cfg(feature = "halo2")]
+    #[strum(serialize = "halo2")]
+    Halo2,
+}
```

Comment:
What else do you foresee going into this `common` crate?

Reply:
The way we do things now (create the proof directly without serializing the constants/witnesses) we need this enum in a number of crates. I'm not sure what else would go in here, maybe the `number` crate?